src/test/java/org/folio/search/controller/InstanceControllerTest.java

```diff
@@ -74,6 +74,25 @@ void search_positive() throws Exception {
       .andExpect(jsonPath("$.instances", is(emptyList())));
   }
 
+  @Test
+  void search_offset_limit_10k() throws Exception {
+
+    var cqlQuery = "title all \"test-query\"";
+
+    when(searchService.search(searchServiceRequest(Instance.class, cqlQuery))).thenReturn(searchResult());
+
+    var requestBuilder = get("/search/instances")
+      .queryParam("query", cqlQuery)
+      .queryParam("limit", "100")
+      .queryParam("offset", "10000")
```

Comment:
try this case 
```
limit = 500 (allowed)
offset = 9600
```

Reply:
at the moment max allowed value for limit is 500, I need to just change values in test to these?

Reply:
in this case test is failed